Source/UTagsEd/Private/desktop.ini

```diff
@@ -0,0 +1,4 @@
+[LocalizedFileNames]
```

Comment:
What is desktop.ini? Is it required, does it have to be in /Source, not /Config?